### PR TITLE
Bg 34 add header

### PR DESCRIPTION
**Pull request by romelu**

### CODE REVIEW

File: code/js/localization/en.json [hunk 1]
```diff
@@ -3,12 +3,34 @@
...
+  "nav.button.contact": "Contact",
+
+  "nav-language-switcher-button": "Language",
+  "nav-language-switcher-english": "English",
+  "nav-language-switcher-czech": "Czech",
+
+  "index.title": "Home",
+  "index.h2": "Hi, I'm Roman Hvizdak",
+  "index.h3": "I present my site where the calculator is located, a page with generating examples for practice and the possibility to print a set of examples",
+  "index.square.p": "More about this project on GitHub",
```
Comment (palomino7) [inline code/js/localization/en.json]: What is it: index.square.p ??? Be more concrete also for all headlines h1,h2 ...

File: code/html/calculator.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width , initial-scale=1">
+  <script src="../js/rcalc-loader.js"></script>
+  <link rel="stylesheet" id="id-page-style" type="text/css" href="../css/theme-default.css">
+  <title data-i18n="calculator.title">Calculator</title>
+</head>
+
```
Comment (palomino7) [inline code/html/calculator.html]: Bad formatting. You don't use some formatter in atom? 
 head and body are inside html ... should be spaced 

in all html files

File: code/js/rcalc-sandwich.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+  $(".rcalc-nav-sandwich").on("click", function() {
+    $("header nav ul").toggleClass("open");
```
Comment (palomino7) [inline code/js/rcalc-sandwich.js]: no strings in js files

File: code/js/localization/en.json (hunk 1 above)
Comment (romelu) [reply code/js/localization/en.json]: index.square.p renamed to index.square.github.paragraph... and all done for headlines also

File: code/html/calculator.html (hunk 2 above)
Comment (romelu) [reply code/html/calculator.html]: I am using Atom Beautify... change the settings and now it does indent even body and head tags inside html

File: code/js/rcalc-sandwich.js (hunk 3 above)
Comment (romelu) [reply code/js/rcalc-sandwich.js]: constants applied... declared in same file, hope its ok if not comment please...

**Pull request merged**